qutebrowser/browser/commands.py

```diff
@@ -1175,6 +1175,25 @@ def tab_move(self, index: typing.Union[str, int] = None, count=None):
         cmdutils.check_overflow(new_idx, 'int')
         self._tabbed_browser.tabBar().moveTab(cur_idx, new_idx)
 
+    @cmdutils.register(instance='command-dispatcher', scope='window')
+    @cmdutils.argument('choice', completion=miscmodels.suggest)
+    def suggest(self, command: str, columns: int,
+                sep_suggestions: str, sep_columns: str,
+                suggestions: str, choice: str):
+        """Allows userscripts to let a user choose between extracted information.
+
+        Args:
+            command: Qutebrowser command to run with the choice
+            columns: self-explanatory
+            sep_suggestions: charsequence which separates suggestions
```

Comment:
maybe these could be flags with a reasonable default (`sep_suggestions=';',sep_columns=',')?
It might be even simpler to use a single separator character and double it to represent a new row.
e.g. `foo,bar,baz,,biz,buz,baa`

Reply:
Actually I guess completion functions can't parse flags yet, but I think that would be the way to go once that is implemented